lambda_function/requirements.txt

```diff
@@ -3,3 +3,5 @@ hermes_spani @ git+https://github.com/HERMES-SOC/hermes_spani.git
 hermes_eea @ git+https://github.com/HERMES-SOC/hermes_eea.git
 hermes_nemisis @ git+https://github.com/HERMES-SOC/hermes_nemisis.git
 hermes_merit @ git+https://github.com/HERMES-SOC/hermes_merit.git
+cdftracker @ git+https://github.com/HERMES-SOC/CDFTracker.git
```

Comment:
Is this going to be used both in sorting and processing? If so should it go in the AWS Utils?

Reply:
Nope just Processing function